tests/post_training/conftest.py

```diff
@@ -87,6 +99,20 @@ def wrapped_get_value(cls, info: Dict[PipelineType, RunInfo], target_pipeline_ty
 
         return wrapped_get_value
 
+    @staticmethod
+    def na_msg(func):
+        """
+        Replace return value of function from None to NOT_AVAILABLE_MESSAGE.
+        """
+
+        def wrapped_na_msg(*args, **kwargs):
```

Comment:
What is the point to have this function?

Reply:
`        Replace return value of function from None to NOT_AVAILABLE_MESSAGE.` in result.csv